presto-testing/src/main/java/io/prestosql/testing/AbstractTestQueryFramework.java

```diff
@@ -115,6 +116,11 @@ protected Object computeScalar(@Language("SQL") String sql)
         return computeActual(sql).getOnlyValue();
     }
 
+    protected void assertMatch(String actual, String regex)
+    {
+        assertTrue(actual.matches(regex), "[" + actual + "] does not match [" + regex + "]");
```

Comment:
the other day I left a comment to use assertj, it has some native support for regex matching. If you apply that comment then I would expect this code become very fluent and so we might have no longer a need for this method (we could just inline it).

assertMatch -> assertMatches